src/views.lisp

```diff
@@ -102,17 +104,21 @@
                                  (#\ü #\ù)
                                  (#\ÿ)
                                  (#\$)))))
-    (map 'string #'(lambda (char)
-                     (let* ((corruptions-for-character (cassoc char corruptions)))
-                       (if (and corruptions-for-character
-                                (eql (random 2)
-                                     1))
-                           (nth (random (length corruptions-for-character)) corruptions-for-character)
-                           char)))
+    (map 'string
+         #'(lambda (char)
+             (let* ((corruptions-for-character (cassoc char corruptions)))
+               (if (and corruptions-for-character
+                        (eql (random 2)
+                             1))
+                   (nth (random (length corruptions-for-character))
+                        corruptions-for-character)
+                   char)))
          text)))
 
 (defun stylize-text-post (post)
-  (acons :data (corrupt (cassoc :data post)) (acons :style (text-style (cassoc :board post)) post)))
```

Comment:
Avoid long one liners like this, use some newlines and indention to add clarity to the structure.